src/lib/memory/homeobject.hpp

```diff
@@ -32,11 +32,18 @@ struct std::hash< homeobject::BlobRoute > {
 
 namespace homeobject {
 
+using btree = std::unordered_map< BlobRoute, Blob >;
+
+struct ShardIndex {
+    mutable std::shared_mutex _btree_lock;
+    btree _btree;
+    blob_id _shard_seq_num{0ull};
```

Comment:
shard_id here?

Reply:
it's "a blob_id" which represents the SSN. blob_id is the type.

Reply:
yes although blob_id is also uint64_t I mean `using shard_id uint64_t` probabaly make it cleaner.  Nevermind